apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c

```diff
@@ -0,0 +1,145 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <pthread.h>
+#include <sys/time.h>
+#include <errno.h>
+#include <apps/shell/tash.h>
+#include <wifi_manager/wifi_manager.h>
+#include "../../../tc_common.h"
+
+void ocftest_wifi_sta_connected(void);
+void ocftest_wifi_sta_disconnected(void);
+void ocftest_wifi_softap_sta_join(void);
+void ocftest_wifi_softap_sta_leave(void);
+
+static wifi_manager_cb_s wifi_callbacks = {
+	NULL,						//ocftest_wifi_sta_connected,
+	NULL,						//ocftest_wifi_sta_disconnected,
+	NULL,						//ocftest_wifi_softap_sta_join,
+	NULL,						//ocftest_wifi_softap_sta_leave,
+};
+
+void utc_wifi_manager_init_n(void)
```

Comment:
all utc functions are used in this file, so let's change to static

Reply:
Good recommendation!. thanks.